oca_event_badge/views/event_event_template_asset_report.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright 2018 ACSONE SA/NV
+     Copyright 2019 Tecnativa - Cristina Martin R.
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 
 <odoo>
 
-    <template id="assets_report" inherit_id="web.report_assets_common">
+    <template id="assets_common" inherit_id="web.report_assets_common">
         <xpath expr="." position="inside">
-          <link href="/oca_event_badge/static/src/css/event_event_template.css" rel="stylesheet"/>
+          <link href="/oca_event_badge/static/src/css/event_event_template.scss" rel="stylesheet" type="text/scss"/>
```

Comment:
Why rename it if there are no parametric styles?

Reply:
It's because in v12 the css became scss, so I considered was better to keep the trend.

Reply:
Well, looking to https://sass-lang.com/documentation/file.SCSS_FOR_SASS_USERS.html, it's valid, but not sure if it's worth to be processed by SCSS compiler. @Yajo, what do you think?